### DISCUSSION TITLE
spinner is interfering with logger

### DISCUSSION THREAD

**Discussion by telatin** [Textualize/rich#1361] [Q&A] [answered]
I just started using Rich (fantastic <3) so apologise if the question is trivial.

I have a CLI script that uses a rich Console and a python logger (standard library) that uses a `RichHandler()`.

I have concurrent jobs that I'd like to decorate with a spinner, so i started with:
```
with console.status("Getting databases", spinner="point"):
   do_work()
```

My proble is that `do_work()` invokes the logger and will also print messages (`logger.error(message, extra={"markup": True})`), but this will not work nicely with the spinner as `console.print()` does. 

Is it possible to use both in a better way? Thanks

**Comment by willmcgugan** [comment, marked as answer]
Pass the console object to your RichHandler

**Reply by telatin** [reply]
Super fast, thanks a million!
